libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState } from 'react'
+import cn from 'classnames'
+import {
+  default as ReactDatePicker,
+  registerLocale,
+  ReactDatePickerProps,
+} from 'react-datepicker'
+import { format, getYear } from 'date-fns'
+import pl from 'date-fns/locale/pl'
+import is from 'date-fns/locale/is'
+import en from 'date-fns/locale/en-US'
+
+import Icon from '../Icon/Icon'
+import Typography from '../Typography/Typography'
+
+import * as styles from './DatePicker.treat'
+import 'react-datepicker/dist/react-datepicker.css'
+import './overwriteStyles.css'
+
+type Locale = 'is' | 'pl'
```

Comment:
does english come out of the box or?

Reply:
Yes, english is the default and it comes out of the box.